src/main/java/com/walkhub/walkhub/global/annotation/WalkhubService.java

```diff
@@ -0,0 +1,16 @@
+package com.walkhub.walkhub.global.annotation;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Transactional(readOnly = true)
+@Service
+public @interface WalkhubService {
```

Comment:
이거 이름
```suggestion
public @interface ServiceWithTransactionalReadOnly {
```
이런 미친 직관성 어떱니까

Reply:
너무 길어... 맨 앞에 거시기는 뭐야